app.js

```diff
@@ -0,0 +1,92 @@
+const billAmount = document.querySelector(".bill-amount");
+const tips = document.querySelectorAll(".tip");
+const customTip = document.querySelector(".custom");
+const numberOfPeople = document.querySelector(".people-tip");
+const tipAmount = document.querySelector(".total-tip");
+const totalTip = document.querySelector(".total");
+const reset = document.querySelector("button");
+
+let totalBill = 0;
+let numberOfPersons = 1;
+let tip = 0;
+let tipsArray = Array.from(tips);
+
+const billValueHandler = (e) => {
+  totalBill = Number(e.target.value);
+  calculateTipAmount();
+};
+
+const numberOfPeopleHandler = (e) => {
+  numberOfPersons = Number(e.target.value);
+  calculateTipAmount();
+};
+
+const tipsHandler = (e) => {
+  tipsArray.forEach((element) => {
+    element.addEventListener("click", () => {
+      tipsArray.forEach((item) => {
+        if (item !== element) {
+          item.classList.remove("active");
+          customTip.value = "";
+        }
+      });
+      element.classList.add("active");
+
+      switch (element.textContent) {
```

Comment:
I think this switch could be either reduced to 2-3 lines or transformed into a function that calculates the tip value based on a number parameter (so it can be reused in the custom tip function). If you think about it, you could just parse the `element.textContent` number value and remove the '%' symbol. For example, when you select lets say 15%, you cut the '%' character and tip value will be 15 / 100. This can be reused for any other percentage.

Reply:
Done!